scripts/scenarios/cloud6/cloud7-upgrade-non-disruptive-ceph.yml

```diff
@@ -93,6 +93,7 @@ proposals:
         admin_keyring: "/etc/ceph/ceph.client.admin.keyring"
         pool: volumes
         user: cinder
+        secret_uuid: 75ffb452-2478-4c4c-abcc-1be39cb6325a
```

Comment:
I'm fine with adapting this PR with the empty value and merging it so we can see if the ceph job accepts it. Of course we can test it manually, I can do it once I have time